### PR TITLE
cms testcase

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: aytests/cms.vars [hunk 1]
```diff
@@ -0,0 +1 @@
+KEYS_URL=http://{{IP}}:{{PORT}}/static/keys
```
Comment (mchf) [inline aytests/cms.vars:1]: Where do IP and PORT come from?

Comment (schubi2) [reply aytests/cms.vars:1]: https://github.com/yast/autoyast-integration-test/blob/master/lib/aytests/image_builder.rb#L365

File: aytests/cms.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e -x
+
+# checking if the keys have been downloaded
+grep "dummy" /etc/salt/pki/minion/minion.pem
+grep "dummy" /etc/salt/pki/minion/minion.pub
```
Comment (mchf) [inline aytests/cms.sh:7]: the test contains ```default.key``` and ```default.pub```. I know nothing about salt, so should I expect that these two files are these ```minion.*``` files?

File: aytests/cms.xml [hunk 3]
```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url>http://download.opensuse.org/repositories/systemsmanagement:/saltstack:/products/SLE_12_SP2/</media_url>
```
Comment (mchf) [inline aytests/cms.xml:8]: the test is being merged for SP3, so why is ```SLE_12_SP2``` in the url?

**Comment by mchf** [comment]
Several "expert" questions only. Mostly caused by lack of knowledge ;-)

File: aytests/cms.sh (hunk 2 above)
Comment (schubi2) [reply aytests/cms.sh:7]: Yes

File: aytests/cms.vars (hunk 1 above)
Comment (mchf) [reply aytests/cms.vars:1]: nice, thanks

File: aytests/cms.xml (hunk 3 above)
Comment (schubi2) [reply aytests/cms.xml:8]: Because there is currently no SP3 branch for that OBS project. We will switch if it is available.

Comment (mchf) [reply aytests/cms.xml:8]: hm, to many places which needs to be edited once we switch to sp3. May be there should be a ```PRODUCT``` variable?

**Review by mchf — APPROVED** [review]
lgtm

**Pull request merged**